README.md

```diff
@@ -1 +1,649 @@
+
+# 0. Abstract
+
+Authorized Wire Authenticated Key Exchange (AWAKE) is an [AKE](https://en.wikipedia.org/wiki/Authenticated_Key_Exchange) built on top of the [UCAN auth token](https://github.com/ucan-wg/spec). AWAKE is similar to other [mutual authentication](https://en.wikipedia.org/wiki/Mutual_authentication) schemes (such as self-signed [mTLS](https://datatracker.ietf.org/doc/html/rfc8705)), but with a focus on authorization and proof. AWAKE leverages the UCAN capability chain to prove access to some resource, validating that the requestor is communicating with a party capable of performing certain actions. This is a helpful root of trust with a well defined context when establishing a secure communications channel.
+
+## Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+# 1 Introduction
+
+AWAKE bootstraps a secure session on top of a public channel. Key exchanges for point-to-point communication are plentiful, but in open, trustless protocols, rooting trust can be a barrier for ad hoc communications channels. Two common approaches are to use a trusted certificate authority, or ignore the principal and "merely" establish a point-to-point channel.
+
+Capability-based systems have a helpful philosophy towards a third path. By emphasizing authorization over authentication, they provide a way to know something provable about what the other party "can do", even if they have no sure way of knowing "who they are". One way of phrasing this is that such an agent is "functionally equivalent to the principal in this context". AWAKE makes use of authorization to bootstrap point-to-point sessions that are both secure and mutually trusted.
+
+## 1.1 Terminology
+
+This document contains shorthand (especially in diagrams) and nuanced senses of some terms. Below is a dictionary of AWAKE-specific terms:
+
+| Term      | Meaning                                              |
+| --------- | ---------------------------------------------------- |
+| Attacker  | An attacker attempting to gain access to the channel |
+| ECDH      | Elliptic Curve Diffie-Hellman                        |
+| PK        | Public key                                           |
+| Requestor | The agent opening the session                        |
+| Responder | The agent being contacted by the Requestor           |
+| SK        | Secret (private) key                                 |
+
+## 1.2 Payload Fields
+
+Payloads are encoding agnostic, but JSON is RECOMMENDED. For JSON, any fields that contain non-JSON values (such as ECDH public keys and encryption payloads) MUST be serialized as unpadded [Base64](https://datatracker.ietf.org/doc/html/rfc4648).
+
+Messages that a peer cannot parse SHOULD be ignored.
```

Comment:
Yeah, I can remove that line. The `RECOMMENDED` is there because it's not a _hard_ requirement (e.g. maybe you store-and-forward unreadable messages). Mainly I wanted to convey that it's fine to ignore messages that you can't parse because they're either not meant for you or malicious.

Would you just remove this line?

Reply:
Makes sense! I'd probably either remove it, or change it to `MUST`. Someone can still do whatever they want with the messages, outside of the flow of the protocol, but they shouldn't be processed as part of AWAKE.